### PR TITLE
add proper color for a:hover code

### PR DESCRIPTION
**Pull request by bogas04**
Fix for https://github.com/nodejs/new.nodejs.org/issues/168

screen shot 2015-09-17 at 10 38 51 pm
becomes
screen shot 2015-09-17 at 10 40 31 pm

### CODE REVIEW

File: layouts/css/base.styl [hunk 1]
```diff
@@ -30,6 +30,8 @@ a:active
 a:hover
     color #fff
     background-color $node-green
+    code
```
Comment (phillipj) [inline layouts/css/base.styl]: A little nit picking, but separating the ordinary `a:hover` styling from the `code` element style improves readability IMO.

ATM this results in a stylus parser error as you forgot to add `color` before the hex below :smile_cat:

Comment (bogas04) [inline layouts/css/base.styl]: oh my bad ! Achievement Unlocked, mess-up simple CSS fix

**Comment by bogas04** [comment]
@phillipj I guess I can't mess it up anymore 🙈 let's merge it

**Comment by fhemberger** [comment]
Thanks!

**Pull request merged**